gr-blocks/grc/xmlrpc_client.block.yml

```diff
@@ -14,18 +14,18 @@ parameters:
 -   id: callback
     label: Callback
     dtype: raw
-    default: set_
+    default: None
 -   id: variable
     label: Variable
     dtype: raw
 
 templates:
     imports: from xmlrpc.client import ServerProxy
-    make: ServerProxy('http://${addr}:${port}')
+    make: ServerProxy('http://'+${addr}+':${port}')
     callbacks:
-    - ${callback}(${variable})
+    - set_${(callback)}(${variable})
```

Comment:
The parentheses here don't do anything, right?

Reply:
That’s correct. I left them there in case there was a reason I didn’t know about.